src/components/List/Separator/Separator.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
```

Comment:
can we place this component together with the other list items (/List/ListItem/Variants)? If you rebase on develop the folder should be there already.